test/chargebeex/customer_test.exs

```diff
@@ -119,6 +119,45 @@ defmodule Chargebeex.CustomerTest do
     end
   end
 
+  describe "stream_list" do
```

Comment:
It would be nice to have a test with offset 1 or 2 to make sure that we get all the resources